app/src/androidTest/java/org/simple/clinic/drugs/PrescriptionRepositoryAndroidTest.kt

```diff
@@ -129,4 +129,34 @@ class PrescriptionRepositoryAndroidTest {
     val storedPrescription = database.prescriptionDao().getOne(correctedPrescription.uuid)!!
```

Comment:
Can we reword the commit message?

Ex: "Add method for updating the prescribed drug duration"